pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: http_mock_adapter
 description: HTTP mock adapter for Dio & Mockito. By simply defining requests and corresponding responses through predefined adapters, you will be able to mock requests sent via Dio.
-version: 0.1.1
+version: 0.1.2
```

Comment:
is it correct?

Reply:
Seems correct as 0.1.2 is the latest published version as far as I know.